demisto_sdk/commands/common/tools.py

```diff
@@ -1105,6 +1107,14 @@ def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignor
                     return FileType.INCIDENT_FIELD
                 if _id.startswith('indicator'):
                     return FileType.INDICATOR_FIELD
+                else:  # not an incident or an indicator
+                    if 'definitionId' in _dict and 'genericModuleId' in _dict:
+                        if 'cliName' in _dict:
+                            return FileType.GENERIC_FIELD
```

Comment:
let's move this before the recognition of Incident fields.
Let's check that the `definitionId` is not `incident` or `indicator` and if so this should be classified as Generic Field